staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -135,6 +141,10 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&deprecatedMasterServiceNamespace, "master-service-namespace", deprecatedMasterServiceNamespace, ""+
 		"DEPRECATED: the namespace from which the kubernetes master services should be injected into pods.")
 
+	fs.IntVar(&s.MaxRequestsPerUserInFlight, "max-requests-per-user-inflight", s.MaxRequestsPerUserInFlight, ""+
+		"The maximum number of requests per one user in flight at a given time. When the server exceeds this, "+
+		"it rejects requests. Zero for no limit.")
```

Comment:
what is the default? could you document the default in the above flag?

Reply:
It seems `pflag` shows default values in flag description only for non-zero defaults, so there is no default value shown for `max-requests-per-user-inflight` due to it's default is `0`. Other flags with zero default value have no explicit "default" in their description.